test/unit/services/test-get-organisational-hierarchy-tree.js

```diff
@@ -0,0 +1,112 @@
+      var organisationalHierarchyTree =
+      proxyquire('../../../app/services/organisational-hierarchy-tree', {'./data/get-organisational-hierarchy-data': getOrganisationalHierarchyData})
+
+      getOrganisationalHierarchyData.resolves(dataHelper.ORGANISATIONAL_HIERARCHY_DATA_SINGLE_BRANCH)
+      organisationalHierarchyTree.build().then(function () {
+        expect(organisationalHierarchyTree.get()).to.eql(dataHelper.ORGANISATIONAL_HIERARCHY_TREE_SINGLE_BRANCH)
+        done()
+      })
+    })
+
+    it('should build a tree with mulitple individuals on a single team', function (done) {
+      var getOrganisationalHierarchyData = sinon.stub()
+      var organisationalHierarchyTree =
+      proxyquire('../../../app/services/organisational-hierarchy-tree', {'./data/get-organisational-hierarchy-data': getOrganisationalHierarchyData})
+
+      getOrganisationalHierarchyData.resolves(dataHelper.ORGANISATIONAL_HIERARCHY_DATA_MULTIPLE_INDIVIDUALS)
+      organisationalHierarchyTree.build().then(function () {
+        expect(organisationalHierarchyTree.get()).to.eql(dataHelper.ORGANISATIONAL_HIERARCHY_TREE_MULTIPLE_INDIVIDUALS)
+        done()
+      })
+    })
+
+    it('should build a tree with mulitple branches', function (done) {
+      var getOrganisationalHierarchyData = sinon.stub()
+      var organisationalHierarchyTree =
+      proxyquire('../../../app/services/organisational-hierarchy-tree', {'./data/get-organisational-hierarchy-data': getOrganisationalHierarchyData})
+
+      getOrganisationalHierarchyData.resolves(dataHelper.ORGANISATIONAL_HIERARCHY_DATA_MULTIPLE_BRANCHES)
+      organisationalHierarchyTree.build().then(function () {
+        expect(organisationalHierarchyTree.get()).to.eql(dataHelper.ORGANISATIONAL_HIERARCHY_TREE_MULTIPLE_BRANCHES)
+        done()
+      })
+    })
+
+    it('should build a tree with null values for nullable fields', function (done) {
+      var getOrganisationalHierarchyData = sinon.stub()
+      var organisationalHierarchyTree =
+      proxyquire('../../../app/services/organisational-hierarchy-tree', {'./data/get-organisational-hierarchy-data': getOrganisationalHierarchyData})
+
+      getOrganisationalHierarchyData.resolves(dataHelper.ORGANISATIONAL_HIERARCHY_DATA_NULL_VALUES)
+      organisationalHierarchyTree.build().then(function () {
+        expect(organisationalHierarchyTree.get()).to.eql(dataHelper.ORGANISATIONAL_HIERARCHY_TREE_NULL_VALUES)
+        done()
+      })
+    })
+
+    it('should throw an error when the DB call throws an error', function (done) {
+      var getOrganisationalHierarchyData = sinon.stub()
+      var organisationalHierarchyTree =
+      proxyquire('../../../app/services/organisational-hierarchy-tree', {'./data/get-organisational-hierarchy-data': getOrganisationalHierarchyData})
+
+      getOrganisationalHierarchyData.rejects('DB Error')
+      organisationalHierarchyTree.build().then(function () {
+        fail()
```

Comment:
I was wrong that this was a thing. I was thinking of this: http://chaijs.com/api/assert/#method_fail. So we need to replace `fail()` with `assert.fail()`